src/components/molecules/todo/TodoForm/TodoForm.container.jsx

```diff
@@ -1,13 +1,14 @@
 import React, { useState } from 'react';
-import { connect } from 'react-redux';
+import { useDispatch } from 'react-redux';
 import { todoAdd } from 'store/todo/todo.duck';
 import TodoForm from './TodoForm';
 
-const TodoFormContainer = ({ onAdd, ...props }) => {
+const TodoFormContainer = props => {
   const [text, setText] = useState('');
```

Comment:
trying out the suggestion feature:
```suggestion
  const dispatch = useDispatch();
  const [text, setText] = useState('');
```